packages/connection/src/class-manager.php

```diff
@@ -50,20 +50,22 @@ class Manager {
 	 *
 	 * @todo Implement a proper nonce verification.
 	 */
-	public function init() {
-		$this->setup_xmlrpc_handlers(
+	public static function configure() {
+		$manager = new self();
```

Comment:
Is this what you're referring to when you say "more valid"? I like it

Reply:
If by "this" you mean initializing the Manager in one place where it makes more sense than in Jetpack's constructor, then yes, that's what I'm referring to :)